gitreceive/receiver/flynn-receive.go

```diff
@@ -189,23 +199,25 @@ Options:
 			Processes: map[string]int{"web": 1},
 		}
 
-		watcher, err := client.WatchJobEvents(app.ID, release.ID)
-		if err != nil {
-			log.Fatalln("Error streaming job events", err)
-			return
-		}
-		defer watcher.Close()
```

Comment:
Initial push:

```
-----> Creating release...
=====> Waiting for web job to start...
-----> new web job is starting
-----> new web job is up
=====> Default web formation scaled to 1
=====> Application deployed
```

Subsequent push:

```
-----> Creating release...
-----> new web job is starting
-----> new web job is up
-----> old web job is stopping
-----> old web job is down
=====> Application deployed
```

I have omitted the job IDs because they are not added to deployment events (maybe they should be), and I don't think we need timestamps either.

WDYT?